at_client/lib/src/service/notification_service_impl.dart

```diff
@@ -93,22 +93,24 @@ class NotificationServiceImpl
           'monitor is already started for ${_atClient.getCurrentAtSign()}');
       return;
     }
-    if (AtClientManager.getInstance()
-        .atClient
-        .getPreferences()!
-        .fetchOfflineNotifications) {
-      final lastNotificationTime = await _getLastNotificationTime();
-      await _monitor!.start(lastNotificationTime: lastNotificationTime);
-    } else {
-      await _monitor!.start();
-    }
+
+    await _monitor!.start(lastNotificationTime: await _getLastNotificationTime());
 
     if (_monitor!.status == MonitorStatus.started) {
       _isMonitorPaused = false;
     }
   }
 
+
   Future<int?> _getLastNotificationTime() async {
+    if (AtClientManager.getInstance().atClient.getPreferences()!.fetchOfflineNotifications == false) {
+      // Don't fetch notifications which were received by the server while this client was offline ...
+      // ... means ask only for notifications received after RIGHT NOW
+      return DateTime.now().millisecondsSinceEpoch;
```

Comment:
I was wondering if the difference in now() if client and server could be a problem. Not passing any value is understood as after now() by the server.

Reply:
OK this whole PR is wrong then 🤡

Need to do a much deeper look at what's going on. What Colin & Kevin observed is that when they set this new flag to "false" then the client would not subsequently sync.

I'll close this PR and create another ticket.